src/accounts/default/set-default-command.js

```diff
@@ -25,6 +34,13 @@ class SetWorkspaceAccountCommand extends Command {
       [commandRegex],
       'direct_message,direct_mention',
       (bot, message) => {
+        this.reportCommandWithCompanyName({
+          userObject: message,
+          teamConfigurationService: this.teamConfigurationService,
+          logger,
+          eventName: 'set-default-account',
+          action: 'triggered the set default account command'
+        });
```

Comment:
[Line 22](https://github.com/logzio/slack-integration/pull/109/files?file-filters%5B%5D=.js#diff-76dbe80c012232a3652541ac3041fd088db35760a51195cafd9d5ab73d0c88e0R22)
same log for different actions(are they?), is that ok?

Reply:
yeah, cuz it's the same command but with alias (more param)